cumulusci.yml

```diff
@@ -162,8 +172,10 @@ flows:
             6:
                 task: deploy_k12_kit_settings
             7:
-                task: execute_install_apex
+                task: add_picklist_values
```

Comment:
agree that it should really be add_application_type_picklist_values (or something similar) because eventually we will have something like add_behavior_type_picklist_values as well.